src/mixins/document-mixin.ts

```diff
@@ -104,7 +111,7 @@ export default class DocumentMixin extends Vue {
       if ((error as any).name === 'PasswordException') {
         return { isEncrypted: true, isContentLocked: true }
       }
+      return null // invalid pdf file
```

Comment:
Previously, if there was an exception in the try block, this method returned a valid object. Now, it returns a null object, which the parent can check to see if there was an error.

(This seemed simpler than re-throwing the error and having another try-catch block (or then/catch) in the parent.)